cypress/integration/immutableDatabaseTests/dropdown.ts

```diff
@@ -252,6 +252,74 @@ describe('Dropdown test', () => {
       // One pending org request should be visible
       cy.get('#my-pending-org-card-0').should('be.visible');
     });
+
+    it('Should be able to delete pending/reject org request', () => {
+      // Both pending and rejected orgs should be visible
+      cy.get('#my-pending-org-card-0').should('be.visible');
+      cy.get('#my-rejected-org-card-0').should('be.visible');
+
+      // New mocked memberships after deleting the rejected organization
+      const memberships = [
+        { id: 1, role: 'MAINTAINER', accepted: false, organization: { id: 1000, status: 'PENDING', name: 'orgOne' } },
+        { id: 2, role: 'MAINTAINER', accepted: true, organization: { id: 1001, status: 'PENDING', name: 'orgTwo' } }
+      ];
+
+      // Route all DELETE API calls to organizations respond with with an empty JSON object
+      cy.server().route({
+        method: 'DELETE',
+        url: '*/organizations/*',
+        response: []
+      });
+
+      // Route GET API call to user/membership with the new mocked membership JSON object
+      cy.server().route({
+        method: 'GET',
+        url: '*/users/user/memberships',
+        response: memberships
+      });
+
+      // Delete the rejected organization
+      // Should result with the rejected organization no longer existing, and only the pending org existing
+      cy.get('#delete-my-rejected-org-0')
+        .should('be.visible')
+        .click();
+      cy.contains('div', 'Are you sure you wish to delete this rejected organization: orgThree?').within(() => {
+        cy.contains('button', 'Delete').click();
+      });
+      cy.get('#my-pending-org-card-0').should('be.visible');
+      cy.get('#my-rejected-org-card-0').should('not.exist');
+
+      // New membership JSON object after deleting the pending organization
+      const memberships2 = [{ id: 1, role: 'MAINTAINER', accepted: false, organization: { id: 1000, status: 'PENDING', name: 'orgOne' } }];
```

Comment:
You don't have to change this, but if you made `{ id: 1, ...` a const, then you could use it here and online 263 without repeating it.

Also I would give it a more meaningful name, like membershipsAfterDeletion.

Reply:
Done. Made the first membership a constant, and renamed the memberships to have more meaningful names.